### PR TITLE
Upgrade to 3.0 Roslyn packages to get better C# 8 support

### PR DESCRIPTION
**Pull request by rchande**

### CODE REVIEW

**Comment by filipw** [comment]
can you add a note to release notes?

File: tests/OmniSharp.Roslyn.CSharp.Tests/IntellisenseFacts.cs [hunk 1]
```diff
@@ -282,7 +282,7 @@ public MyClass2()
                 ";
 
             var completions = await FindCompletionsAsync(filename, source);
-            ContainsCompletions(completions.Select(c => c.CompletionText).Take(1), "text:");
+            ContainsCompletions(completions.Select(c => c.CompletionText).Take(1), "text");
```
Comment (mholo65) [inline tests/OmniSharp.Roslyn.CSharp.Tests/IntellisenseFacts.cs]: Shouldn’t the actual completion be `text:` here?

Comment (rchande) [reply tests/OmniSharp.Roslyn.CSharp.Tests/IntellisenseFacts.cs]: This test started failing after I updated Roslyn--I guess this is a behavior change on their side.

Comment (mholo65) [reply tests/OmniSharp.Roslyn.CSharp.Tests/IntellisenseFacts.cs]: Yes, I figured. Don’t know the internals of OmniSharp completion handler, but it feels like we need to figure out if this is a bug in Roslyn or if we are ”doing it wrong”. AFAIK, this is a named parameter, thus `:` should be required?

**Pull request merged**